conmon-rs/server/src/terminal.rs

```diff
@@ -322,6 +322,12 @@ impl AsRawFd for TerminalFd {
     }
 }
 
+impl AsFd for TerminalFd {
+    fn as_fd(&self) -> BorrowedFd<'_> {
```

Comment:
Nit: the lifetime can be omitted:
```suggestion
    fn as_fd(&self) -> BorrowedFd {
```